### PR TITLE
The v2.0 releases

### PR DESCRIPTION
**Pull request by nkartyshov**
https://github.com/AdguardTeam/ContentBlocker/issues/94
https://github.com/AdguardTeam/ContentBlocker/issues/96
https://github.com/AdguardTeam/ContentBlocker/issues/97
https://github.com/AdguardTeam/ContentBlocker/issues/98

### CODE REVIEW

File: adguard_cb/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -38,14 +38,14 @@
     <!--About Adguard-->
     <string name="versionInfoTextViewText">الإصدار {0}</string>
     <string name="copyrightInfoTextViewText">© 2008–2017. Performix LLC.\nجميع الحقوق محفوظه.</string>
```
Comment (ameshkov) [inline adguard_cb/src/main/res/values-ar/strings.xml]: 2008 --> 2009
Performix LLC --> Adguard Software Ltd.

**Comment by nkartyshov** [comment]
Implemeted task https://github.com/AdguardTeam/ContentBlocker/issues/82
Minor fixed

File: adguard_cb/src/main/java/com/adguard/android/service/PreferencesService.java [hunk 2]
```diff
@@ -239,4 +163,8 @@ This file is part of Adguard Content Blocker (https://github.com/AdguardTeam/Con
     void incBrowserConnectedCount();
 
     int getBrowserConnectedCount();
+
+    boolean isShowAboutOtherProduct();
```
Comment (ameshkov) [inline adguard_cb/src/main/java/com/adguard/android/service/PreferencesService.java]: is/setWelcomeMessage()

Also, add a comment

File: adguard_cb/src/main/java/com/adguard/android/service/PreferencesServiceImpl.java [hunk 3]
```diff
@@ -37,29 +36,23 @@ This file is part of Adguard Content Blocker (https://github.com/AdguardTeam/Con
...
 
     private final static Logger LOG = LoggerFactory.getLogger(PreferencesServiceImpl.class);
 
     private final static String KEY_AUTOUPDATE_FILTERS = "auto_update_filters";
     private final static String KEY_UPDATE_OVER_WIFI = "update_over_wifi";
-    private final static String KEY_SEND_ANONYMOUS_STATISTICS = "help_browsing_security_enabled";
-    private final static String KEY_LAST_VERSION_FOUND = "last_version_found";
     private final static String KEY_LAST_UPDATE_CHECK_DATE = "last_update_check_date";
-    private final static String KEY_LOG_LEVEL = "log_level";
     private final static String KEY_REFERRER = "referrer";
```
Comment (ameshkov) [inline adguard_cb/src/main/java/com/adguard/android/service/PreferencesServiceImpl.java]: Do we need it? Please remove all the unnecessary keys

File: adguard_cb/src/main/java/com/adguard/android/contentblocker/MainActivity.java [hunk 4]
```diff
@@ -123,11 +125,34 @@ public void onClick(View v) {
...
+                    bottomBarView.setVisibility(View.GONE);
+                }
+            });
+
+            bottomBarView.findViewById(R.id.learn_more).setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    preferencesService.setShowAboutOtherProduct(true);
+                    bottomBarView.setVisibility(View.GONE);
+                    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("http://agrd.io/cb_adguard_products")));
```
Comment (ameshkov) [inline adguard_cb/src/main/java/com/adguard/android/contentblocker/MainActivity.java]: Please use Chrome CustomTabs instead

**Comment by nkartyshov** [comment]
I fixed the problem and made a small refactoring.

File: adguard_cb/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -514,6 +515,8 @@
                 </LinearLayout>
 
             </ScrollView>
+
+           <include layout="@layout/botton_bar"/>
```
Comment (Revertron) [inline adguard_cb/src/main/res/layout/activity_main.xml]: bottoM

**Comment by nkartyshov** [comment]
Fixed

**Pull request merged**